### PR TITLE
Deploying faucet on kubernetes

### PR DESCRIPTION
**Pull request by 3h4x**
- [x] GHA to build docker images and push to ECR
- [x] GHA to deploy faucet
- [x] change links in our documentation and other places to point to new faucet url faucet.t0rn.io 
- [x] run inside container as unprivileged user

Faucet works and connects to correct endpoint.
Screenshot 2023-01-10 at 18 02 41

### CODE REVIEW

File: api/package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "AMOUNT": 10000000000000,
     "DECIMALS": 12,
     "SS58_PREFIX": 42,
-    "PROVIDER": "ws://localhost:9933",
+    "PROVIDER": "ws://ws.t0rn.io:1933",
```
Comment (chiefbiiko) [inline api/package.json]: If we go thru public DNS let's use `wss://`

**Comment by chiefbiiko** [comment]
Do we wanna have `t0rn.io` as a base domain (or sub to `t3rn.io`)?

File: api/package.json (hunk 1 above)
Comment (3h4x) [reply api/package.json]: True, true. Fixed

**Comment by 3h4x** [comment]
> Do we wanna have `t0rn.io` as a base domain (or sub to `t3rn.io`)?

I actually don't know 100%. IMO it's better separation, which can help in improving security and mitigate deployment issue with services that requires DNS.
Here I used t0rn.io because it's available within our K8S test cluster.
DNS t3rn.io is not in our AWS account so without migration or delegation it's problematic.

@AwesomeIbex what are your thoughts?

**Review by carrion256 — APPROVED** [review]

**Pull request merged**